Refactoring

Robin1711 opened the pull request:
Refactoring of css, html and codeShow.js. Also applying comments of Mircea

Robin1711 committed:
minor fixes

Robin1711 committed:
major refactoring css and html

Robin1711 committed:
refactoring html and css

yarchiT committed:
Add hint to code show, refactor context

yarchiT committed:
Refactor settings page

Robin1711 committed:
Fixed message and added comments

Robin1711 committed:
remove console.logs

Robin1711 committed:
remove console.log

yarchiT committed:
Merge branch 'context' into refCodeShow

Nickborchers commented on config.xml:
```diff
@@ -1,19 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
`<tizen:category name="com.samsung.wmanager.WATCH_CLOCK"/>`

This line should still be in this file, as discussed with Mircea and Niels

Nickborchers commented on index.html:
```diff
@@ -6,7 +6,7 @@
+    <title>Time to Learn</title>
 
     <link rel="stylesheet" type="text/css" href="css/style.css" />
    <!-- <script data-main="js/build/main-built.js" src="js/lib/require.js"></script> -->
```
` <script data-main="js/build/main-built.js" src="js/lib/require.js"></script> `
Should also be here, uncommented, since the version on Github should be deployable

Nickborchers commented on js/app/codeShow.js:
```diff
@@ -83,82 +79,81 @@ define(['userData', 'qrcode'], function(userData, qrcode) {
-	function initQRCodePage() {
-		printTitle("Your 8 digit code:", "qrcodeHeaderCanvas");
-		placePageIcon(2);
+/*	The function initQRCode will create a QR Code and will store this QR Code in "imageQR". The QRCode is only rendered once. */
```
Nicely documented, very clear.

Nickborchers commented on js/app/codeShow.js:
```diff
@@ -83,82 +79,81 @@ define(['userData', 'qrcode'], function(userData, qrcode) {
-	function activateBackButtons() {
+
+/*	The function InitPage will initialize all the sections of the page. */
+	function initPage() {
```
Nice

Nickborchers commented on js/app/codeShow.js:
```diff
@@ -83,82 +79,81 @@ define(['userData', 'qrcode'], function(userData, qrcode) {
 		changePage: function() {
-			page = (page === 1? 2 : 1);
-			activateScreen(page);
+			page = (page === TEXTUAL_PAGE? QRCODE_PAGE : TEXTUAL_PAGE);
```
Good refactoring

Nickborchers reviewed (changes requested):
No real issues spotted, apart from the two lines which should be uncommented and/or added in config.xml and index.html, so fix that and we can merge

Robin1711 committed:
update confit.xml

Added <tizen:category name="com.samsung.wmanager.WATCH_CLOCK"/>

Robin1711 committed:
Update index.html

uncommented <script data-main="js/build/main-built.js" src="js/lib/require.js"></script>
